components/shared/upload/DropUpload.vue

```diff
@@ -91,7 +91,7 @@ const props = withDefaults(
     icon: 'cloud-arrow-up',
     preview: false,
     expanded: false,
-    accept: '',
+    accept: '.zip',
```

Comment:
as you correctly mentioned this will not work as it is a shared component
this change will break minting

Reply:
yeah I just set the default value to zip, it can still support other values

Reply:
sorry i didn't notice you have created a new shared DropUpload component
overall we need here a component that will allow to upload zip and check that the contents are supported image formats.

I don't see it as shared functionality

unless you make
1. a shared component that will allow zip uploads
2. validate the zip contents in massmint

I personally would just create the required component in massmint and not bother trying to make it shared at the moment

Reply:
can we create a follow up issue for these and unzip and merge this pr as its getting huge

Reply:
Agree

Reply:
created followup #5328